pro/src/pages/CollectiveOfferStockCreation/CollectiveOfferStockCreation.tsx

```diff
@@ -37,18 +37,17 @@ const CollectiveOfferStockCreation = ({
   ) => {
     let isOk: boolean
     let message: string | null
-    let payload: { id: string } | null
+    let createdOfferTemplateId: string | undefined
```

Comment:
juste pour le débat,
pour dire `| undefined` tu pense quoi de : 

```suggestion
    let createdOfferTemplateId?: string
```
?

Reply:
Ce n'est pas exactement la même chose mais j'aurai tendance à dire que si on veut initialiser une valeur vide il faut utiliser null et non undefined.
`let mavar = undefined` n'a pas vraiment de sens
`let mavar = null` c'est ok
Du coup je préfèrerai `let createdOfferTemplateId?: string` ou `let createdOfferTemplateId: string | null` :slightly_smiling_face:

Reply:
Et pour départager encore plus je dirais `let createdOfferTemplateId: string | null` car quand on déclare une variable ça n'a pas de sens de la laisser undefined, ce type est censé être utilisé pour checker si une var existe ou non

![image](https://user-images.githubusercontent.com/6317823/202662866-1f990d46-7a68-4e18-85c7-05ef4b14cb44.png)

Reply:
Merci pour les précisions je note pour les prochaines fois 👍